setup.py

```diff
@@ -44,16 +44,16 @@ def find_package_data(where='.', package='', exclude=standard_exclude, exclude_d
 
 setup(name='docassemble.ALLouisianaSC',
       version='0.0.4',
-      description=("A package customized for Louisiana, specifically the Supreme Court"),
-      long_description="# docassemble.ALLouisianaSC\r\n\r\nA package customized for Louisiana, specifically the Supreme Court\r\n\r\n## Author\r\n\r\nAmanda Brown\r\n\r\n",
+      description=('A package customized for Louisiana, specifically the Supreme Court'),
+      long_description='# docassemble.ALLouisianaSC\r\n\r\nA package customized for Louisiana, specifically the Supreme Court\r\n\r\n## Author\r\n\r\nAmanda Brown\r\n\r\n',
       long_description_content_type='text/markdown',
       author='System Administrator',
       author_email='admin@admin.com',
       license='The MIT License (MIT)',
       url='https://docassemble.org',
       packages=find_packages(),
       namespace_packages=['docassemble'],
-      install_requires=['docassemble-AssemblyLine'],
```

Comment:
Worth it to keep this installed, if things need to ever be spun up from scratch again?

Reply:
I think it should be docassemble.AssemblyLine, no? Maybe this relates to @amandaleighbrown's current package install challenges?

Reply:
I guess I just found it odd that the test server was having all these issues but the production server worked just fine. Either way, it's fixed! Thank you!